fsd_utils/services/aws.py

```diff
@@ -0,0 +1,347 @@
+                    )
+        except ClientError:
+            print(f"Couldn't delete message from queue {queue_url}")
+        else:
+            return response
+
+    def create_sqs_queue(
+        self, queue_name, has_dlq=False, dlq_queue_name=None, max_recieve_count=3
+    ):
+        """
+        Creates an Amazon SQS & DLQ queue.
+
+        :param queue_name: SQS queue name to create
+        :param has_dlq: If true, DLQ(Dead letter queue) is created for given queue
+        :param dlq_queue_name: If `has_dlq` is true, then DLQ queue name to create
+        :param max_recieve_count: If `has_dlq` is true, then provide the max recieve count
+                                  before moving messages to DLQ
+        :return: (sqs_queue_url) URL of the queue created.
+        """
+        # get queue list
+        queue_list = self.get_queues()
+
+        # create SQS queue if not exists
+        if queue_name not in queue_list:
+            sqs_queue_url = self.client.create_queue(
+                QueueName=queue_name,
+            )["QueueUrl"]
+            print(f"Successfully created SQS queue '{queue_name}'")
+        else:
+            print(f"SQS queue '{queue_name}' already exists!")
+            sqs_queue_url = self.client.get_queue_url(QueueName=queue_name)["QueueUrl"]
+
+        if has_dlq:
+            # create DLQ queue if not exists
+            if dlq_queue_name not in queue_list:
+                dlq_queue_url = self.client.create_queue(
+                    QueueName=dlq_queue_name,
+                )["QueueUrl"]
+                dlq_queue_arn = self.client.get_queue_attributes(
+                    QueueUrl=dlq_queue_url, AttributeNames=["QueueArn"]
+                )["Attributes"]["QueueArn"]
+            else:
+                print(f"DLQ '{dlq_queue_name}' already exists!")
+                dlq_queue_url = self.client.get_queue_url(QueueName=dlq_queue_name)[
+                    "QueueUrl"
+                ]
+                dlq_queue_arn = self.client.get_queue_attributes(
+                    QueueUrl=dlq_queue_url, AttributeNames=["QueueArn"]
+                )["Attributes"]["QueueArn"]
+
+            redrive_policy = {
+                "deadLetterTargetArn": dlq_queue_arn,
+                "maxReceiveCount": max_recieve_count,
+            }
+
+            self.client.set_queue_attributes(
+                QueueUrl=sqs_queue_url,
+                Attributes={"RedrivePolicy": json.dumps(redrive_policy)},
+            )
+
+        return sqs_queue_url
+
+    def set_queue_attributes(
+        self, queue_name: str = None, queue_url: str = None, attributes: dict = {}
+    ):
+        if not queue_url:
+            queue_url = self.client.get_queue_url(QueueName=queue_name)["QueueUrl"]
+
+        self.client.set_queue_attributes(
+            QueueUrl=queue_url,
+            Attributes=attributes,
+        )
+
+
+if __name__ == "__main__":
```

Comment:
Is this for debugging purposes? Can it be removed?

Reply:
Yes but also is an example of SQS client usage. I have commented it out now. Any developer want to learn the usage can simply uncomment & run the code